install/operator/build/conf/config.yaml

```diff
@@ -13,7 +14,7 @@ Images:
     OAuthProxy: oauth-proxy
     OAuthProxyImage: origin-oauth-proxy
     PostgresExporter: postgres_exporter
-    Postgresql: postgresql
+    Postgresql: postgresql-95-rhel7
```

Comment:
this image doesn't exist in docker.io/rhscl , so no DB will be spawned when using community build with this config

Reply:
Solved with https://github.com/syndesisio/syndesis/pull/7255/commits/f1a3e8b10d5f8de930ff1bd5a26cb780f43368e7